mini.go

```diff
@@ -129,15 +137,8 @@ func GET(writer http.ResponseWriter, reader *http.Request) {
 		return
 	}
 
-	//output the raw bytes to the browser
-	writer.WriteHeader(http.StatusOK)
-	io.WriteString(writer, string(bytes))
-
 	// log the event
-	log.Println("GET:", c.ID)
-
-	//log the time
-	log.Println("GET: took", time.Now().Sub(starttime), "to execute")
+	wrapper.info(c.ID, c.Type, "GET successful", c.SiteID, time.Now().Sub(starttime))
```

Comment:
not going to log time anymore? if so, drop the var `starttime` (and in fact this should give a compile error for unused var)